libs/common-services/src/main/java/uk/gov/di/ipv/core/library/domain/VerifiableCredential.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.di.ipv.core.library.domain;
+
+import com.nimbusds.jwt.JWTClaimsSet;
+import com.nimbusds.jwt.SignedJWT;
+import lombok.Getter;
+import uk.gov.di.ipv.core.library.exceptions.CredentialParseException;
+import uk.gov.di.ipv.core.library.persistence.item.VcStoreItem;
+
+import java.text.ParseException;
+
+@Getter
+public class VerifiableCredential {
+    private final String userId;
```

Comment:
Do you think this is a useful field to have? It looks like we only really use it in a couple of places where we could source the user ID elsewhere.

Note that this is _almost_ always the same as `claimsSet.getSubject()` - but won't be for inherited identity VCs.

Reply:
I think that it's defined by how we store it in our VcStore - which makes it natural to pair with criId. I think even in the inherited case we'll store it by claimsSet.getSubject()? But we won't validate with that userId, instead we could just skip since the check isn't relevant